tests/tools/oesign/test-inputs/invalid_syntax_1.conf

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Enclave settings:
+# invalid syntax - "debug"
+
+debug=1      #should be Debug=1
```

Comment:
Side discussion: @anakrish @mingweishih I don't know if this was a design choice or an arbitrary implementation choice, should oesign be case insensitive with respect to labels? The current behavior is fine since most input formats are case sensitive (e.g. JSON, XML ...)

Reply:
@anakrish @mingweishih , since this is a side discussion, i will first mark this comment as resolved. and you can put your comments when you review this PR.